src/server/components/dashboard/relatedLinks/relatedLinkSchema.ts

```diff
@@ -0,0 +1,34 @@
+import Joi, { type CustomHelpers } from "joi";
```

Comment:
I wish the code in line **1** worked but it looks like it doesn't :(

<img width="919" alt="Screenshot 2023-06-06 at 14 21 26" src="https://github.com/UKForeignOffice/lists/assets/1377253/07507488-b9d5-43ab-b3a2-2217a41414b6">